models/thoth-support-github-ttm/ThothGitHubTTMModel.py

```diff
@@ -0,0 +1,107 @@
+"""Here goes the prediction code."""
+import os
+import joblib
+import boto3
+from io import BytesIO
+from typing import Iterable, Dict, List, Union
+
+import numpy as np
+import pandas as pd
+
+
+class ThothGitHubTTMModel(object):
+    """
+    Model template.
+
+    You can load your model parameters in __init__ from a location accessible at runtime.
+    """
+
+    def __init__(self):
+        """
+        Add any initialization parameters.
+
+        These will be passed at runtime from the graph definition parameters
+        defined in your seldondeployment kubernetes resource manifest.
+        """
+        print("Initializing")
+        self.model = self.load_model_from_s3()
+
+    def load_model_from_s3(self):
+        """Load pretrained model from ceph s3 bucket."""
+        # init s3 client
+        s3_resource = boto3.resource(
+            "s3",
+            endpoint_url=os.getenv("S3_ENDPOINT"),
+            aws_access_key_id=os.getenv("S3_ACCESS_KEY"),
+            aws_secret_access_key=os.getenv("S3_SECRET_KEY"),
+        )
+
+        # download into a buffer
+        buffer = BytesIO()
+        s3_object = s3_resource.Object(
+            os.getenv("S3_BUCKET"),
+            f"{os.getenv('S3_MODEL_KEY')}/model.joblib",
+        )
+        s3_object.download_fileobj(buffer)
+
+        # load from buffer
+        model = joblib.load(buffer)
+        return model
+
+    def class_names(self) -> Iterable[str]:
+        """Return names of output classes."""
+        return [
+            f"{i}"
+            for i in [
+                "0-1 mins ",
+                "1-2 mins",
+                "2-5 mins",
+                "5-10 mins",
+                "10-30 mins",
+                " < 2 hours",
+                "2-8 hours",
+                "8-17 hours",
+                " < 4 days",
+                " > 4 days",
+            ]
+        ]
```

Comment:
```suggestion
        return [
            "0-1 mins",
            "1-2 mins",
            "2-5 mins",
            "5-10 mins",
            "10-30 mins",
            " < 2 hours",
            "2-8 hours",
            "8-17 hours",
            " < 4 days",
            " > 4 days",
        ]
```
can directly return the list too :)